assets/js/blocks/mini-cart/style.scss

```diff
@@ -19,6 +19,8 @@
 }
 
 .wc-block-mini-cart__amount {
+	font-size: inherit;
+	font-weight: inherit;
```

Comment:
If this is being hidden via the `display: none;` rule, does it still make sense to set the font size and weight?